app/build.gradle.kts

```diff
@@ -89,6 +89,7 @@ dependencies {
     implementation(project(":ui-kit"))
     implementation(project(":core"))
     implementation(project(":ethereumkit"))
+    implementation(project(":logger-crashlytics"))
```

Comment:
i thought this module is responsible only for crashlytics lib, but inside there is sentry too. 
maybe we can rename it? or don't extract it at all?